deploy/helm/sumologic/templates/chart-configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sumologic-configmap
```

Comment:
do we need an entirely new configmap, or could we just put this data into the existing configmap?

Reply:
I did not add this in the `collection-sumologic` config map as that is primarily for fluentd config.

Reply:
I'm confused, a configmap can store more than one key...

And what about a map called `collection-sumologic` is specific to fluentd?

Reply:
spoke offline, since we're already creating separate configmaps for each deployment, and this one spans across deployments, I'm ok with a separate configmap.

In the future we may want to consolidate all these configmaps though to reduce complexity of the chart.